docs/man-pages/condor_history.rst

```diff
@@ -1,42 +1,191 @@
 *condor_history*
-=================
+================
 
-View log of HTCondor jobs completed to date
-:index:`condor_history<single: condor_history; HTCondor commands>`
-:index:`condor_history command`
+View historical ClassAds.
+
+:index:`condor_history<double: condor_history; HTCondor commands>`
 
 Synopsis
 --------
 
-**condor_history** [**-help** ]
-
-**condor_history** [**-name** *name*]
-[**-pool** *centralmanagerhostname[:portnumber]*] [**-backwards**]
-[**-forwards**] [**-constraint** *expr*] [**-file** *filename*]
-[**-userlog** *filename*] [**-search** *path*] [**-dir | -directory**]
-[**-local**] [**-startd**] [**-epochs**]
-[**-format** *formatString AttributeName*]
-[**-autoformat[:jlhVr,tng]** *attr1 [attr2 ...]*]
-[**-l | -long | -xml | -json | -jsonl**] [**-match | -limit** *number*]
-[**-attributes** *attr1[,attr2...]*]
-[**-print-format** *file* ] [**-wide**]
-[**-since** *time_or_jobid* ] [**-completedsince** *time_expr*] [**-scanlimit** *number*]
-[**cluster | cluster.process | owner**]
+**condor_history** [**-help**]
+
+**condor_history** [*OPTIONS*] [*cluster*... | *cluster.proc*... | *user*...]
+
+**condor_history** [**-debug**] [**-name** *name*] [**-pool** *hostname[:port]*]
+[**-backwards** | **-forwards**] [**-constraint**] [**-since** *jobid | expr*]
+[**-completedsince** *timestamp*] [**-scanlimit** *N*] [**-match | -limit** *N*]
+[**-local**] [**-schedd** | **-startd** | **-epochs[:d]**] [**-file** *filename*]
+[**-userlog** *filename*] [**-search** *path*] [**-dir/-directory**]
+[**-format** *FormatString* *Attribute*] [**-af/-autoformat[:jlhVr,tng]** *Attribute [Attribute ...]*]
+[**-print-format** *filename*] [**-l/-long**] [**-attributes** *Attribute[,Attribute,...]*]
+[**-xml**] [**-json**] [**-jsonl**] [**-wide[:N]**]
 
 Description
 -----------
 
-*condor_history* displays a summary of all HTCondor jobs listed in the
-specified history files. If no history files are specified with the
-**-file** option, the local history file as specified in HTCondor's
-configuration file (``$(SPOOL)``/history by default) is read. The
-default listing summarizes in reverse chronological order each job on a
+Display summaries of all historical ClassAds listed in history files.
+By default the local history file specified by :macro:`HISTORY` is read
+for ClassAds of completed HTCondor Jobs.
```

Comment:
Should we same something here about how *condor_history* can optionally query a remote schedd here?